virtcontainers/clh_test.go

```diff
@@ -0,0 +1,718 @@
+// Copyright (c) 2019 Ericsson Eurolab Deutschland G.m.b.H.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/kata-containers/runtime/virtcontainers/device/config"
+	"github.com/kata-containers/runtime/virtcontainers/store"
+	"github.com/kata-containers/runtime/virtcontainers/types"
+	"github.com/stretchr/testify/assert"
+)
+
+//
+// Cli helper functions
+//
+
+func cliToString(args []string) string {
```

Comment:
You could simplify this (or remove it?) in favour of:

```go
strings.TrimSpace(strings.Join(args, " "))
```

Reply:
removed and renamed